RocketChat/Rocket.Chat
# feat: Add Sticky Date for Improved Message Date Visibility

## Proposed changes

Added a sticky date feature to improve user experience in Rocket.Chat. Previously, users had to scroll to the top to identify the date of a message. With this enhancement, the date now remains sticky at the top, providing users with a quick reference point for message dates.

## Screenshots or Videos

[screencast-www.google.com-2023.12.23-11_55_32.webm](https://github.com/RocketChat/Rocket.Chat/assets/74670675/756fa2e1-3afb-4cc5-81f1-cac2d8f34954)

## Steps to test or reproduce

1. Open a chat.
2. Scroll through messages.
3. Observe the sticky date at the top for easy reference.





## Further comments

This enhancement addresses the issue where users had difficulty identifying message dates. The sticky date feature is implemented to enhance user convenience by providing a persistent reference point for message dates.
